components/Home/Home.jsx

```diff
@@ -27,17 +29,87 @@ const welcomeName = 'Shriya';
 
 export default () => {
   useEffect(() => {
-    // This is where we would hit our database, but for now we'll have fake data
-    console.log('Home: Initializing Workout store');
-    dispatch({ type: INITIALIZE_WORKOUTS, workouts: DBWorkoutResponse });
+    const initializeDatabase = async () => {
+      // Get the current logged in user id
+      const currentUser = firebase.auth().currentUser.uid;
+      // const currentUser = '68w6wWz8l5QJO3tDukh1fRXWYjD2';
```

Comment:
Forgot to mention this, but when you're running this locally before Fin's task to update firebase when a user is created, comment out line 34 and uncomment line 35 and you'll be able to get the test data in my personal account